src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -305,42 +397,43 @@ paths:
         '200':
           description: A Metadata object describing this service.
           schema:
-              $ref: '#/definitions/Metadata'
-
+            $ref: '#/definitions/Metadata'
   /toolClasses:
     get:
       summary: List all tool types
-      description: >
+      description: |
         This endpoint returns all tool-classes available
       tags:
         - GA4GH
       responses:
         '200':
-          description: An array of methods that match the filter.
+          description: A list of potential tool classes.
```

Comment:
Array might be better since it's a swagger keyword that is mentioned right below